container_main.py

```
@@ -66,19 +66,36 @@ def run_scheduler():
+            "--workers",
+            "2",
+            "--timeout",
+            "120",
+            "--access-logfile",
+            "-",
+            "--error-logfile",
+            "-",
+            "api_server:app",
+        ]
+
+        logger.info(f"Executing: {' '.join(cmd)}")
```

Ruff (0.15.0)

[warning] Logging statement uses f-string

(G004)